src/playercards/PlayerCardPanel.ttslua

```diff
@@ -0,0 +1,509 @@
+require("playercards/PlayerCardPanelData")
+local spawnBag = require("playercards/spawnbag/SpawnBag")
+local arkhamDb = require("arkhamdb/ArkhamDb")
+
+-- TODO: Update when the real UI image is in place
+local BUTTON_WIDTH = 150
+local BUTTON_HEIGHT = 550
+
+local ALL_CARDS_BAG_GUID = "15bb07"
+
+local FACE_UP_ROTATION = { x = 0, y = 270, z = 0}
+local FACE_DOWN_ROTATION = { x = 0, y = 270, z = 180}
+
+-- Coordinates to begin laying out cards to match the reserved areas of the
+-- table.  Cards will lay out horizontally, then create additional rows
+local START_POSITIONS = {
+  skill = Vector(58.384, 1.36, 92.4),
+  event = Vector(53.229, 1.36, 92.4),
+  asset = Vector(40.960, 1.36, 92.4),
```

Comment:
Should probably round the values to two decimals

Reply:
I've been going to three (unless the value I'm going for falls very close to a different one) because that's the standard we use for rounding with ModManager.  In this case, the values were taken directly from the card spreads, which were taken from the previous memory bags, to align with the decals.  Less an issue once we remove them, but broadly speaking I think as long as we're matching to that 3-place rounding, this should be a no-comment.